frontend/include/chpl/resolution/scope-types.h

```diff
@@ -22,17 +22,22 @@
 
 #include "chpl/types/Type.h"
 #include "chpl/uast/AstNode.h"
+#include "chpl/uast/Decl.h"
 #include "chpl/util/memory.h"
 #include "chpl/util/iteration.h"
 
 #include <unordered_map>
 #include <utility>
 
+#include "llvm/ADT/Optional.h"
+
 namespace chpl {
 namespace resolution {
 
 class BorrowedIdsWithName;
 
+using OwnedID = std::pair<ID, uast::Decl::Visibility>;
```

Comment:
Could we call this 'IdAndVis` or `OwnedIdAndVis` or something? I perosnally don't think the `Owned` part of the name is relevant.